code/modules/clothing/gloves/thick.dm

```diff
@@ -23,19 +23,26 @@
 
 /obj/item/clothing/gloves/thick/swat
 	desc = "These tactical gloves are somewhat fire and impact-resistant."
-	name = "\improper SWAT Gloves"
-		
+	name = "improper SWAT Gloves"
+	material = /decl/material/solid/cloth
+
 /obj/item/clothing/gloves/thick/combat //Combined effect of SWAT gloves and insulated gloves
 	desc = "These tactical gloves are somewhat fire and impact resistant."
 	name = "combat gloves"
 	siemens_coefficient = 0
 	permeability_coefficient = 0.05
+	material = /decl/material/solid/cloth
+	matter = list(
+		/decl/material/solid/metal/steel = MATTER_AMOUNT_PRIMARY,
```

Comment:
`MATTER_AMOUNT_PRIMARY` is meant for whatever is in the `material` slot, should really be using `MATTER_AMOUNT_REINFORCEMENT` at most for sub-materials.